src/modules/sidebar/sidebar.component.scss

```diff
@@ -17,24 +26,33 @@
     &:hover {
       background-color: $stache-color-gray-light;
     }
-
-    &.active,
-    &.active:hover {
-      background-color: $stache-color-blue;
-      color: #fff;
-    }
   }
 
-  .stache-nav-sidebar .stache-nav-sidebar {
-    background-color: $stache-color-gray-light;
-    padding: 0; 
-    margin-bottom: 8px;
+  .stache-nav-sidebar {
+    .stache-nav-sub {
+      background-color: $stache-color-gray-light;
+      padding: 0; 
+      margin-bottom: 8px;
+      display: none;
```

Comment:
Cool. I figured, I just didn't see where that CSS was manipulated in the sidebar nav component.